suites/suites/suites.yaml

```diff
@@ -336,28 +336,6 @@ handler_configurations:
       <<: *lab_openstack_inputs
       keystone_tenant_name: system-tests/system-test-4
 
-  lab_openstack_system_tests4_secured:
```

Comment:
There should also be a diff that removes/modifies the suites that uses `lab_openstack_system_tests4_secured` handler configuration, no?
I mean, I suppose you want modify the existing suite definition + add a new test group with the security test